test/e2e/tests/consensus_test.go

```diff
@@ -0,0 +1,55 @@
+package e2e_test
```

Comment:
This file should be named `evidence_test.go`, I think, since we're testing evidence recording and propagation.